### PR TITLE
{ServiceBus} `az servicebus namespace private-endpoint-connection approve/reject/delete/show/list` `az servicebus namespace private-link-resource show`: Migrating servicebus namespace private-endpoint-connection cmdlets to AAZ

### PR DESCRIPTION
**Pull request by schaudhari6254888**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ServiceBus

File: src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_endpoint_connection/__cmd_group.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "servicebus namespace private-endpoint-connection",
+)
+class __CMDGroup(AAZCommandGroup):
+    """servicebus namespace private-endpoint-connection
+    """
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_endpoint_connection/__cmd_group.py:19]: Maybe we can have some meaningful description for command group help msg?

File: src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_link_resource/__cmd_group.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "servicebus namespace private-link-resource",
+)
+class __CMDGroup(AAZCommandGroup):
+    """private-link-resource
+    """
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_link_resource/__cmd_group.py:19]: same here

File: src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_endpoint_connection/__cmd_group.py (hunk 1 above)
Comment (schaudhari6254888) [reply src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_endpoint_connection/__cmd_group.py:19]: Done.

File: src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_link_resource/__cmd_group.py (hunk 2 above)
Comment (schaudhari6254888) [reply src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_link_resource/__cmd_group.py:19]: Done

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**